flashcards/web/src/modules/AddDeck/ImagePicker.tsx

```diff
@@ -0,0 +1,204 @@
+import { Container, styled } from "@mui/material";
+import { useEffect, useState } from "react";
+import { faker } from "@faker-js/faker";
+import PreviewImageLibrary from "./PreviewImageLibrary";
+import ModalImageLibrary from "./ModalImageLibrary";
+
+const StyledInput = styled("div")({
```

Comment:
Same problem, styles is available as part of MUI however tailwind was chosen as styling library